### PR TITLE
Feature/proxy restructuring 2

### PR DESCRIPTION
**Pull request by mtrunkat**
@jirimoravcik 
- I made some small adjustments on index.md and usage.md, please see the comments 
- But I think that in order to simplify our lives we should do one large change on following 3 md files (datacenter, residential, and serp pages):
    - Let's extract the shared info from those pages and place everything to usage.md - for example sessions are repeated there over and over and that's not nessesary
    - Let's extract the examples and put them to examples.md after usage.md - this is because examples are basically the same and maintaining them 3x everywhere is just too much work. 
   
Could you look into those 2 changes? I think that it will shrink the size of docs and makes it much easier to maintain.

### CODE REVIEW

File: sources/platform/proxy/datacenter_proxy.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ sidebar_position: 10.3
 slug: /proxy/datacenter-proxy
 ---
 
+import Tabs from '@theme/Tabs';
```
Comment (mtrunkat) [inline sources/platform/proxy/datacenter_proxy.md:8]: Without this the Tabs are not being rendered

File: sources/platform/proxy/index.md [hunk 2]
```diff
@@ -6,23 +6,29 @@ category: platform
...
 You can use proxies in your [actors](../actors/index.mdx) or any other application that supports HTTP proxies. Apify Proxy monitors the health of your IP pool and intelligently [rotates addresses](#ip-address-rotation) to prevent IP address-based blocking.
 
-**You can view your proxy settings and password on the [Proxy](https://console.apify.com/proxy) page in the Apify Console.**
+You can view your proxy settings and password on the [Proxy](https://console.apify.com/proxy) page in the Apify Console. For pricing information, visit [apify.com/pricing](https://apify.com/pricing).
+
 
 ## Quickstart {#quickstart}
 
-If you want to start using Apify Proxy as soon as possible, we provide minimal working examples for connecting to our proxy using the Apify SDK ([JavaScript](/sdk/js) and [Python](/sdk/python)).
+Usage of Apify Proxy means just a couple of lines of code thanks to our SDKs for ([JavaScript](/sdk/js) and [Python](/sdk/python)):
```
Comment (mtrunkat) [inline sources/platform/proxy/index.md:29]: Just keep it simple

File: sources/platform/proxy/index.md [hunk 3]
```diff
@@ -68,13 +73,14 @@ async def main():
...
 if __name__ == '__main__':
     asyncio.run(main())
-
 ```
 
 </TabItem>
-</Tab>
+</Tabs>
+
+## Proxy types {#proxy-types}
```
Comment (mtrunkat) [inline sources/platform/proxy/index.md:82]: Some small explanation

File: sources/platform/proxy/index.md [hunk 4]
```diff
@@ -94,40 +100,3 @@ if __name__ == '__main__':
     />
 </CardGrid>
 
-**For pricing information, visit [apify.com/proxy](https://apify.com/proxy).**
```
Comment (mtrunkat) [inline sources/platform/proxy/index.md:97]: I moved this to "Usage" as it belongs there. We could list it here as features but I don't think it's necessary for the main landing page with explanation

File: sources/platform/proxy/usage.md [hunk 5]
```diff
@@ -11,24 +11,29 @@ slug: /proxy/usage
 
 ## Connection settings
 
-Below are the HTTP proxy connection settings for Apify Proxy.
+To connect to the Apify Proxy, you use the [HTTP proxy protocol](https://en.wikipedia.org/wiki/Proxy_server#Web_proxy_servers). This means that you need to configure your HTTP client to use the proxy server at `proxy.apify.com:8000` and provide it with your Apify Proxy password and the other parameters described below.
+
+The full connection string has the following format:
+
+```text
```
Comment (mtrunkat) [inline sources/platform/proxy/usage.md:18]: First, show the connection string before diving into its components. This is explains it better

File: sources/platform/proxy/usage.md [hunk 6]
```diff
@@ -11,24 +11,29 @@ slug: /proxy/usage
...
-| Username            | Specifies the proxy parameters such as groups, [session](./index.md) and location. <br/>See [username parameters](#username-parameters) below for details. <br/>**Note**: this is not your Apify username.                                                                                                                                |
-| Password            | Proxy password. Your password is displayed on the [Proxy](https://console.apify.com/proxy/groups) page in Apify Console. <br/>In Apify [actors](../actors/index.mdx), it is passed as the `APIFY_PROXY_PASSWORD` <br/>environment variable.<br/>See the [environment variables docs](../actors/development/programming_interface/environment_variables.md) for more details. |
-| Connection URL      | `http://<username>:<password>@proxy.apify.com:8000`                                                                                                                                                                                                                                                                                                        |
```
Comment (mtrunkat) [inline sources/platform/proxy/usage.md:23]: As the connection string is above, it does not need to be repeated here.

**Review by jirimoravcik — APPROVED** [review]
changes look good, I'm working on the other changes in a new branch based off this PR

**Pull request merged**